c/pthread/bounded_buffer_false-unreach-call.c

```diff
@@ -0,0 +1,377 @@
+extern void __VERIFIER_error() __attribute__ ((__noreturn__));
+extern int __VERIFIER_nondet_int(void);
+
+#include <stdio.h>
```

Comment:
Please also add the preprocessed files.
A tool might not support to evaluate macros.